webpack/react_app/components/TargetingHosts/HostItem/HostItem.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ActionButtons } from 'foremanReact/components/common/ActionButtons/ActionButtons';
+import HostStatus from '../HostStatus/HostStatus';
+
+const HostItem = ({ name, link, status, actions }) => {
+  const renderName = () => {
+    if (link !== '' || link !== undefined) {
+      return <a href={link}>{name}</a>;
+    }
+    return (
+      <a href="#" className="disabled">
+        {name}
+      </a>
+    );
+  };
+
+  return (
+    <tr>
+      <td className="host_name">{renderName()}</td>
+      <td className="host_status">
+        <HostStatus status={status} />
+      </td>
+      <td className="host_actions">
+        <ActionButtons buttons={[...actions]} />
```

Comment:
It must be like that because of error:
```
ImmutableError {name: "MyError", message: "The shift method cannot be invoked on an Immutable data structure."}
```